docs/usage.rst

```diff
@@ -92,14 +92,15 @@ importing changed files back in addition to the main FTML format. These filters
 from the File menu of the Main Window.
 
 MSS currently offers several import/export filters in the mslib.plugins.io module, which may serve
-as an example for the definition of own plugins. They are listed below. The CSV plugin is enabled
-by default. Enabling the experimental FliteStar text import plugin would require those lines in
+as an example for the definition of own plugins. Take care that added plugins use different file extensions.
+They are listed below. The CSV plugin is enabled by default.
+Enabling the experimental FliteStar text import plugin would require those lines in
 the UI settings file:
 
 .. code:: text
 
     "import_plugins": {
-        "FliteStar": ["txt", "mslib.plugins.io.flitestar", "load_from_flitestar"]
+        "FliteStar": ["fls", "mslib.plugins.io.flitestar", "load_from_flitestar"]
```

Comment:
I am unsure about this change because the FliteStar tool will use the "txt" extension. This requires a manual rename of the file by the user, which is impossibly difficult for certain users (i.e the OS might scare them out of it).

Reply:
I added it to be consistant with one extension for one type. But anyway it is always only a suggestion.